### PR TITLE
update WSL installation docs for current builds

### PR DESCRIPTION
**Pull request by nitzmahone**
##### SUMMARY
* supersedes #25224

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
winrm.py

##### ANSIBLE VERSION
2.4.0

### CODE REVIEW

File: docs/docsite/rst/intro_windows.rst [hunk 1]
```diff
@@ -47,17 +47,27 @@ If you would like to experiment with the Windows Subsystem for Linux (WSL), firs
...
+
+    # this step is only necessary for Windows builds earlier than 16188, and must be repeated each time bash is launched,
+    # unless bash is launched as ``bash --login``
+    # see https://github.com/Microsoft/BashOnWindows/issues/2148 and
+    # https://github.com/Microsoft/BashOnWindows/issues/816#issuecomment-301216901 for details
+    source ~/.profile
+
+After you've successfully run these commands, you can start to create your inventory, write example playbooks and start targeting systems using the plethora of available Windows modules.
+
+If you want to run Ansible from source for development purposes, simply uninstall the pip-installed version (which will leave all the necessary dependencies behind), then clone the Ansible source, and run the hacking script to configure it to run from source::
```
Comment (jhawkesworth) [inline docs/docsite/rst/intro_windows.rst]: I've not had to remove the pip-installed version of ansible.  So long as you remember to cd into the ansible source tree and run 'source hacking/env-setup' they seem to be able to co-exist.  On balance I think leave as is - these instructions are safer as they avoid you ever getting mixed up between release and development versions.

**Review by jhawkesworth — APPROVED** [review]

File: docs/docsite/rst/intro_windows.rst [hunk 2]
```diff
@@ -47,17 +47,27 @@ If you would like to experiment with the Windows Subsystem for Linux (WSL), firs
 `these instructions <https://www.jeffgeerling.com/blog/2017/using-ansible-through-windows-10s-subsystem-linux>`_.
 This requires a reboot.
 
-Once WSL is enabled, you can open the Bash terminal. The first time you so this, a few questions need to be answered.
-At the prompt you can quickly start using the Ansible devel branch by running the following commands::
+Once WSL is enabled, you can open the Bash terminal. At the prompt, you can quickly start using the latest Ansible release by running the following commands::
```
Comment (petemounce) [inline docs/docsite/rst/intro_windows.rst:50]: nit: Spurious double `:`?

Comment (sivel) [reply docs/docsite/rst/intro_windows.rst:50]: The double `::` is needed to specify that the next "block" is a code block.

Comment (petemounce) [reply docs/docsite/rst/intro_windows.rst:50]: Or is that part of RST? I see this elsewhere in the file.

Comment (petemounce) [reply docs/docsite/rst/intro_windows.rst:50]: Got it; thanks.

File: docs/docsite/rst/intro_windows.rst (hunk 1 above)
Comment (nitzmahone) [reply docs/docsite/rst/intro_windows.rst]: Yeah, there are just enough "weird little things" that can happen (e.g., module/module_util lookup fallthru, finding the wrong version of stuff that moved), especially if the pip version is drastically different than devel, that I err on the side of caution. :)

**Pull request merged**